src/components/notes/cards/Business.tsx

```diff
@@ -0,0 +1,34 @@
+/** Vendor */
+import React from "react";
+
+/** Redux */
+import { IRootState } from "@redux/configureStore";
+import { useSelector } from "react-redux";
+
+/** Custom */
+import Note from "./Note";
+import { INote } from "../../../types/note";
+
+function Business() {
+  const notes = useSelector((state: IRootState) => state.note.list);
+
+  const filteredNotes = notes.filter((note) => note.category === "Business");
```

Comment:
`categories` should be an Enum. For instance under the `types` directory, create a file called `enums.ts` with the following:
```
export enum NoteCategory = {
  Business = "business",
  Another1 = "another_1",
  YetAnother = "yet_another"
};
```

And in the your `types.ts` file export as such `export { NoteCategory } from "./enums"`

And then import to this file as `import { NoteCategory } from "../../../types`. 

Use in your code is as such `notes.filter((note: INote) => note.category === NoteCategory.Business)`

The purpose is to strictly type annotate the options and no deviate from the use of the enum. This ensures you are always using active categories in this case and not ones that are not. Often when a development team works on the same project a lot of ideas get thrown around and the variable sprawl takes afoot. So trying to conduct a filter on a string that only exists in one file might be removed or replaced in favor of anther string option to filter on and you would need to find every occurance of the `business` string logic implementation and replace, often things get missed doing it this way. Also you can combine enums and types together to create an even stronger type system as such:
```
export interface INote {
  ...the other note types
  category: NoteCategory;
}
```

This way for you to build the INote object, it not only has to contain category but the type HAS to be derived from the enum and not a string directly. Attempting a string would throw an error in your typescript code. This helps with Intellisense and writing better code in general. If you have more questions please let me know.